.coveragerc

```diff
@@ -13,3 +13,4 @@ source =
 exclude_lines =
     @abc.abstractmethod
     @abc.abstractproperty
+    @property
```

Comment:
https://codecov.io/github/pyca/cryptography/tests/test_interfaces.py#l-50 (I also see it locally during coverage. I vaguely recall "missed branch" being why we excluded the abc decorators as well)